php/php.nette.tester/test/unit/src/org/netbeans/modules/php/nette/tester/coverage/CloverLogParserTest.java

```diff
@@ -98,7 +95,10 @@ private void fixContent(File file) throws Exception {
         Path path = file.toPath();
         Charset charset = StandardCharsets.UTF_8;
         String content = new String(Files.readAllBytes(path), charset);
-        content = content.replaceAll("%WORKDIR%", getDataDir().getAbsolutePath());
+        String workdirReplacement = Matcher.quoteReplacement(getDataDir().getAbsolutePath());
+        content = content.replaceAll("%WORKDIR%", workdirReplacement);
+        String separatorReplacement = Matcher.quoteReplacement(File.separator);
+        content = content.replaceAll("%SEP%", separatorReplacement);
```

Comment:
And once more - cannot `content.replace("%SEP%", File.separator)` be used?

Reply:
Right, I changed it as you suggest.